docs/manual/troubleshooting.tex

```diff
@@ -570,9 +573,10 @@
   If you encountered the problem in an IDE, reproduce it from the command
   line; if you cannot, report the bug to the IDE integration.
 
-  It can be helpful to add \code{-version -AprintGitProperties -verbose
-    -AprintAllQualifiers} to the javac options.  This causes the compiler
-  to output debugging information, including its version number.
+  It can be helpful to add \\
+  \code{-version -Aversion -AprintGitProperties -verbose -AprintAllQualifiers} \\
```

Comment:
Instead of `-AprintAllQualifiers`, use `-AprintVerboseGenerics` which will also print all qualifiers and verbose generics.